CRFModuleValidation/HeartrateStep.json

```diff
@@ -3,22 +3,23 @@
                            {
                            "identifier"     : "instruction",
                            "type"           : "instruction",
-                           "animatedImage"  : {
+                           "image"          : {
                                                "imageNames" : ["heartRateMeasurement1", "heartRateMeasurement2", "heartRateMeasurement3"],
-                                               "animationDuration" : 2,
-                                               "backgroundColorName" : "macaroniAndCheese"
+                                               "placementType" : "topBackground",
+                                               "animationDuration" : 2
                                                },
+                           "colorTheme"     : { "backgroundColor" : "macaroniAndCheese", "usesLightStyle" : true },
                            "title"          : "Capture heart rate",
                            "text"           : "Use your finger to cover the camera and flash on the back of your phone.",
                            "actions"        : { "goForward": { "iconName": "captureStartButton" }},
-                           "viewInfo"       : { "viewIdentifier": "ActiveInstruction", "storyboardIdentifier": "ActiveTaskSteps" }
+                           "viewTheme"      : { "viewIdentifier": "ActiveInstruction", "storyboardIdentifier": "ActiveTaskSteps" }
                            },
                            {
                            "identifier"   : "heartRate",
                            "type"         : "active",
                            "text"         : "Gently cover both the camera and flash with your finger.",
                            "shouldHideActions": ["goBackward", "goForward"],
-                           "duration": 60,
+                           "duration": 6,
```

Comment:
Is this change intentional, or did you do it to speed up testing and then forgot to change it back?

Reply:
Yes, this is intentional. Thanks for double-checking though.  :)  (The 12 minute run is also intentionally 12 seconds.)